spec/raven/event_spec.rb

```diff
@@ -347,6 +347,7 @@ def raven_context
   end
 
   describe '.initialize' do
+    # NOTE: This test is completely pointless.
```

Comment:
Still, there's RSpec matcher for that.

And the reason it's used is not because you can't `yield` from a block, of course you can and I get that you'll get an exception either way.

The point is that the matcher will:

1st - autogenerate a meaningful test description if you don't manually set one
2nd - make it obvious what you're trying to achieve here.


The point is that this test does not expect anything against the actual `subject`.

`should not touch the env object for an ignored environment` can replace the word `env` with `UFO` and still it would be true, cause we don't expect anything here.